Gaijinizer patch/Map050.txt

```diff
@@ -1172,9 +1185,9 @@ anything.#MTLed
 あなたが気に病む必要はないんだよ。
 > CONTEXT: Map050/events/57/pages/1/157/Dialogue/kazuya
 > CONTEXT: Map188/events/57/pages/1/157/Dialogue/kazuya
-…………"Thank you". Thanks to you, at least they
-should have been at peace. You don't have to
-worry about it.#MTLed
+... "Thank you". Thanks to you, those people at
+least passed on peacefully. You don't need to
+worry about it.#Ron
```

Comment:
Minor formatting and flow:
>_**...Thank you.
Thanks to you, those people at least passed on
peacefully. You don't need to worry about them
any longer...**_